generic/generic-efi.yaml

```diff
@@ -0,0 +1,62 @@
+{{- $architecture := or .architecture "amd64" -}}
+{{- $suite := or .suite "bullseye" -}}
+{{- $kernel := or .kernel (printf "linux-image-%s" $architecture) -}}
+{{if eq $architecture "i386"}}
+    {{ $kernel = or .kernel "linux-image-686" }}
+{{end}}
+{{- $grubtarget := (printf "grub-efi-%s-bin" $architecture) -}}
+{{if eq $architecture "amd64"}}
+    {{ $grubtarget = "x86_64-efi" }}
+{{end}}
+{{ $image := or .image (printf "debian-%s-%s.img" $suite $architecture) }}
+{{- $username := or .username "user" -}}
+{{- $password := or .password "1234" -}}
+
+architecture: {{ $architecture }}
+
+actions:
+  - action: debootstrap
+    suite: {{ $suite }}
+    components:
+      - main
+    mirror: https://deb.debian.org/debian
+
+  - action: apt
+    description: Install packages
+    packages: [ {{ $kernel }}, grub-efi, sudo ]
+
+  - action: run
+    description: Add user
+    chroot: true
+    command: sh -c "adduser --gecos {{ $username }} --disabled-password --shell /bin/bash {{ $username }};
```

Comment:
Debos already wraps the command in `sh -c`, there’s no need to do so once again.

Reply:
You can do this:
```yaml
    command: |
      adduser --gecos {{ $username }} --disabled-password --shell /bin/bash {{ $username }}
      adduser {{ $username }} sudo
      echo "{{ $username }}:{{ $password }}" | chpasswd
```

Reply:
The downside of using `|` is that until https://github.com/go-debos/debos/pull/321 is merged, it’s going to look a bit untidy in the logs.

Reply:
or add `label: foo` to the recipe? :-)